### DISCUSSION TITLE
Sir without root without pc ke phone se bloatware kese uninstall kare  pls sir help

### DISCUSSION THREAD

**Discussion by jackjackraka** [termux/termux-app#2490] [Q&A] [answered]
Sir without root without pc ke phone se bloatware kese uninstall kare 
pls sir help

**Comment by ghost** [comment, marked as answer]
If you're on Android 11. you should be able to use **Wireless Debugging** built in Developer options. check

https://github.com/termux/termux-packages/issues/8218#issuecomment-991999740

Then once setup. you are free to run command
`adb shell pm uninstall -k --user 0 package.name`
